capirca/lib/nsxt.py

```diff
@@ -317,20 +356,22 @@ def _ParseFilterOptions(self, filter_options):
 
   def __str__(self):
     """Render the output of the nsxt policy."""
-    target_header = _RULE_JSON
-    target = []
-
+    # TODO: Handle when there are many policies
```

Comment:
Can you expand on what this would entail? Is it supposed to say "when there are too many policies" instead?